pkg/micropython/doc.txt

```diff
@@ -0,0 +1,74 @@
+/**
+ * @defgroup pkg_micropython MicroPython RIOT port
+ * @ingroup  pkg
+ * @brief    MicroPython - Python for microcontrollers
+ *
+ * # MicroPython RIOT package
+ *
+ * "MicroPython is a lean and efficient implementation of the Python 3
+ *  programming language that includes a small subset of the Python standard
+ *  library and is optimised to run on microcontrollers and in constrained
+ *  environments."
+ *
+ * @see https://micropython.org
+ *
+ * ## Status
+ *
+ * MicroPython on RIOT has to be considered experimental. While the basic
+ * interpreter works fairly well on native and Cortex-M, it has not seen much
+ * testing.
+ *
+ * ## Implementation details
+ *
+ * The RIOT port of MicroPython currently resides in a fork at
+ * https://github.com/kaspar030/micropython (in branch add_riot_port). It is
+ * based on Micropython's "ports/minimal" with some extra modules enabled.
+ * It re-uses the gc_collect code from ports/unix, which has special support
+ * for i386 and Cortex-M. On other platforms, it uses setjmp() to collect
+ * registers.
+ *
+ * ## MicroPython's test suite
+ *
+ * It is possible to run MicroPython's test suite for testing this port.
+ *
+ * Steps:
+ *
+ * 1. flash tests/pkg_micropython
```

Comment:
The Steps can be simplified in simple commands:
- for native
```
make -C examples/micropython all test
```
- on hardware:
```
make BOARD=samr21-xpro -C examples/micropython all test
```

Reply:
No. The RIOT supplied test script tests the RIOT specific modules. The steps are for how to run the upstream test suite, which (currently) requires patching it in most cases, as RIOT's uart is not using flow control by default.

Reply:
Ok, I see. But still, use `make -C examples/micropython flash` instead of `flash ...`. Better be explicit, right ?

Replace `tests/pkg_micropython` with `examples/micropython` in the other items.